src/realmCryo.ttl

```diff
@@ -50,57 +52,133 @@ sorelsp:inside rdf:type owl:ObjectProperty .
+              skos:definition  [
+                rdfs:comment  "A floating mass of ice which is 1) unattached to land or land-fast ice and 2) moved by the action of winds or currents."@en ;
+                dcterms:source <https://orcid.org/0000-0003-4808-4736> ;
+                dcterms:created "2019-12-10T08:45:12-08:00Z"^^xsd:dateTimeStamp ;
+                dcterms:creator <https://orcid.org/0000-0003-4091-6059> ;
+                prov:wasDerivedFrom <http://purl.obolibrary.org/obo/ENVO_01001517> ;
+              ] .
 
 
 ###  http://sweetontology.net/realmCryo/FastIce
 soreac:FastIce rdf:type owl:Class ;
              rdfs:subClassOf soreac:SeaIce ;
-             rdfs:label "fast ice"@en .
-
+             rdfs:label "fast ice"@en ;
+             skos:closeMatch <http://purl.obolibrary.org/obo/ENVO_01001522> ;
+             rdfs:comment "Fast ice is central to the livelihoods of Arctic coastal communities who use it to access fishing and hunting grounds. Vertical fluctuations may be observed during changes of sea level. Fast ice may be formed on site from sea water or by freezing of pack ice of any age to the shore, and it may extend a few yards (meters) or several hundred miles (kilometers) from the coast. Fast ice may be more than one year old and may then be prefixed with appropriate age category (old, second- year, or multiyear). If it is thicker than about 7 ft (2 m) above sea level, it is called an ice shelf."@en ;
+             skos:definition  [
+               rdfs:comment  "A mass of sea ice which 1) is less than 2 meters in thickness and 2) forms along the coast, where it is attached to the shore, to an ice wall, to an ice front, or between shoals or grounded icebergs."@en ;
+               dcterms:source <https://orcid.org/0000-0003-4808-4736> ;
+               dcterms:created "2019-12-10T08:50:11-08:00Z"^^xsd:dateTimeStamp ;
+               dcterms:creator <https://orcid.org/0000-0003-4091-6059> ;
+               prov:wasDerivedFrom <http://purl.obolibrary.org/obo/ENVO_01001522> ;
+             ] .
 
 ###  http://sweetontology.net/realmCryo/Floe
 soreac:Floe rdf:type owl:Class ;
           owl:equivalentClass soreac:IceFloe ;
           rdfs:subClassOf soreac:SeaIce ;
-          rdfs:label "floe"@en .
+          rdfs:label "floe"@en ;
+          skos:closeMatch <http://purl.obolibrary.org/obo/ENVO_03000066> ;
+          skos:definition  [
+            rdfs:comment  "An ice floe which is formed from frozen sea water, and floats upon the surface of a marine water body."@en ;
```

Comment:
This term again has a naming problem - since it is matched to ENVO's "sea ice floe" which does match the consensus definition of all 9 of the GCW glossaries that use the term.  That definition is:

"Any contiguous piece of sea ice".

Reply:
simple solution:
_"A contiguous piece of water ice formed from frozen sea water that floats upon the surface of a marine water body."_

Reply:
Collective comments from our [Semantic Harmonization call](
https://docs.google.com/document/d/1IFA6AsCdxESh9mZB5W1bZ_vARFkF3OvnL-KgdGNcp60/edit#heading=h.2uotl4egsd38)

The definition comes from ENVO, where the superclass of the sea ice floe (mapped above) contains the notion of contiguousness.

Reply:
definition is in pull request, so can't put issue in esipfed sweet.  another issue with complete definition practice in SWEET, with all associated problems...